components/tree-select/interface.tsx

```diff
@@ -1,15 +1,32 @@
 import * as React from 'react';
 import { AbstractSelectProps } from '../select';
 
-export interface TreeData {
-  key: string;
+export type TreeNode = TreeNodeNormal | TreeNodeSimpleMode;
+
+interface TreeNodeNormal {
   value: string;
   /**
-   * @deprecated Please use `title` instead.
-   */
+ * @deprecated Please use `title` instead.
+ */
```

Comment:
why removed space? should be

```javascript
/**
 * @deprecated Please use `title` instead.
 */
```

Reply:
@ilanus sorry, probably my editor settings

Reply:
@ilanus I changed that but CI fails